packages/web3-shared/src/utils/index.ts

```diff
@@ -12,9 +12,9 @@ import {
     Web3Constants,
 } from '../types'
 
-export function isSameAddress(addrA: string, addrB: string) {
-    return addrA.toLowerCase() === addrB.toLowerCase()
-}
+export const isSameAddress = (addrA: string, addrB: string) => addrA.toLowerCase() === addrB.toLowerCase()
+
+export const MatchAddress = (a: string) => (b: string) => isSameAddress(a, b)
```

Comment:
Why it's uppercased?

Reply:
my idea......

Reply:
Can you explain why?

Reply:
https://github.com/DimensionDev/Maskbook/issues/3409

Reply:
I mean why it should be `Match` not `match`